v22.1/changefeed-for.md

```diff
@@ -59,6 +59,7 @@ Option | Value | Description
 `confluent_schema_registry` | Schema Registry address | The [Schema Registry](https://docs.confluent.io/current/schema-registry/docs/index.html#sr) address is required to use `avro`.
 <a name="split-column-families"></a>`split_column_families` | N/A | **New in v22.1:** Target a table with multiple columns families. Emit messages for each column family in the target table. Each message will include the label: `table.family`.
 <a name="virtual-columns"></a>`virtual_columns` | `STRING` | **New in v22.1:** Changefeeds omit [virtual computed columns](computed-columns.html) from emitted [messages](changefeed-messages.html#responses) by default. To maintain the behavior of previous CockroachDB versions where the changefeed would emit [`NULL`](null-handling.html) values for virtual computed columns, set `virtual_columns = "null"` when you start a changefeed. <br><br>You may also define `virtual_columns = "omitted"`, though this is already the default behavior for v22.1+. If you do not set `"omitted"` on a table with virtual computed columns when you create a changefeed, you will receive a warning that changefeeds will filter out virtual computed values. <br><br>**Default:** `"omitted"`
+<a name="min-checkpoint-frequency"></a>`min_checkpoint_frequency` | [Duration string](https://pkg.go.dev/time#ParseDuration) | Controls how often nodes flush their progress to the [coordinating changefeed node](change-data-capture-overview.html#how-does-an-enterprise-changefeed-work). Changefeeds will wait for at least the specified duration before a flushing. This can help you control the flush frequency to achieve better throughput. If this is set to `0s`, a node will flush as long as the high-water mark has increased for the ranges that particular node is processing. If a changefeed is resumed, then `min_checkpoint_frequency` is the amount of time that changefeed will need to catch up. That is, it could emit duplicate messages during this time. <br><br>**Note:** This option's `30s` default will determine when you receive [`resolved`](#resolved-option) messages. That is, if you require `resolved` messages more frequently than `30s`, you must configure `min_checkpoint_frequency` to at least the desired `resolved` message frequency.  <br><br>**Default:** `30s`
```

Comment:
nit: I think we can reshuffle the last two sentences for clarity, something similar to your `resolved` note which I think captures this need very well. How about:

> **Note:** `resolved` messages will not be emitted more frequently than the configured `min_checkpoint_frequency` (but may be emitted less frequently). Since `min_checkpoint_frequency` defaults to `30s`, you **must** configure `min_checkpoint_frequency` to at least the desired `resolved` message frequency if you require `resolved` messages more frequently than `30s`